### DISCUSSION TITLE
Add Option to Minimize Channel Header

### DISCUSSION THREAD

**Discussion by JCRouzer29** [night/betterttv#5992] [Ideas]
The Twitch channel header holds many options with distinct icons.  However when the window size is minimal the text begins to have a significant impact on the overall UI.  The channel header also gives higher priority to the leftmost options pushing some off the right-hand-side, including the viewer counter, uptime, and share options.

This request is to have BTTV add an option to remove the text for these across any and all channels, even removing the button for the React system would be appreciated.  Currently this must be done manually for each page on load and refresh and would greatly appreciate if this could be automated.

If a text alteration instead of outright removal is preferred then consider the following suggestions.
- "Manage Your Sub" -> "Manage"
- "Gift a Sub" -> "Gift"
- "[Re]Subscribe" -> "Sub"

![BTTV Request Reference Image](https://user-images.githubusercontent.com/13194274/222275263-653a767f-fb0c-45ff-ae0c-cf1d42557671.jpg)

**Comment by night** [comment]
Closing this one out — Twitch has redesigned the channel header since this was filed, so the specific buttons and text this references no longer match the current UI. Please open a new discussion if there's a current channel header issue you'd like to flag.
